pkg/gadgets/standardtracerbase.go

```diff
@@ -0,0 +1,90 @@
+// Copyright 2019-2022 The Inspektor Gadget authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package gadgets
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+	"syscall"
+	"time"
+)
+
+/* StandardTracerBase is a type used by gadgets that have a BCC Python-based
```

Comment:
I am fine with `/*` comment but we should be consistent (otherwise I will translate all the comments I wrote to this form :smiling_imp:)

Reply:
I got confused by the style. Fixed it.